backend/webdav/webdav.go

```diff
@@ -86,6 +90,11 @@ func init() {
 			Name:     "bearer_token_command",
 			Help:     "Command to run to get a bearer token",
 			Advanced: true,
+		}, {
+			Name:     "chunk_size",
+			Help:     "Chunk size to use for uploading, in bytes (Nextcloud only)",
```

Comment:
@ncw it would be better if chunk size can be set with a size suffix (eg 100M). How can I do this ?

Reply:
@tcoupin

https://github.com/rclone/rclone/blob/v1.53.4/backend/chunker/chunker.go#L150

https://github.com/rclone/rclone/blob/v1.53.4/backend/chunker/chunker.go#L309

Reply:
thanks